Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang
+
+ADD . /go/src/github.com/salemove/github-review-helper
```

Comment:
`ADD .` changes the intermediate container any time source code changes => this step must be as late as possible to reuse intermediate containers.